ISSUE_TRIAGE.md

```diff
@@ -0,0 +1,276 @@
+Triaging of issues
```

Comment:
Should we link this from the issue templates?

Reply:
Yes, eventually. But one thing at a time, want to get this merged. Then we can start iterating on this and improve other documentation with links and such.